Removed AUDIENCE, upgrade to keycloak 1.7.0.Final -  ERRAI-885

hrstoyanov opened the pull request.

hrstoyanov commented:
Max,
Please merge this simple fix to allow Errai to work with Keycloak 1.7

mbarkley commented on errai-security/errai-security-keycloak/src/main/java/org/jboss/errai/security/keycloak/KeycloakAuthenticationService.java:
```diff
@@ -200,7 +200,7 @@ protected User createKeycloakUser(final AccessToken accessToken) {
     properties.add(new KeycloakProperty(StandardUserProperties.LAST_NAME, accessToken.getFamilyName()));
     properties.add(new KeycloakProperty(StandardUserProperties.EMAIL, accessToken.getEmail()));
-    properties.add(new KeycloakProperty(AUDIENCE, accessToken.getAudience()));
+    //properties.add(new KeycloakProperty(AUDIENCE, accessToken.getAudience()));
```
Please remove this line rather than commenting it out.

mbarkley commented:
Hi Hristo,

Thanks for the PR! Could you please delete the line of commented code in the `KeycloakAuthenticationService`? After that I'll be happy to merge this for you.

Cheers.

hrstoyanov commented:
Max,
I deleted the line. I also updated the BOM to reference Keycloak 1.7.0, although 1.2 will still work. The important change was that AUDIENCE filed that was dropped from KC some while ago, causing runtime errors if someone uses newer KC. Let me know when this is merged into a SHANPSHOT so I can start using it from the official repo.

mbarkley commented:
Hristo,
I've squashed your two commits and cherry-picked them to [master](https://github.com/errai/errai/commit/306e94eb2580b2b53bb6c7819f83d12e47fc13ff) and [3.2](https://github.com/errai/errai/commit/941629a7894aa8460abac6e7f1d02d62e7585c8e).
Cheers.

hrstoyanov commented:
Thanks, I will test them soon! Thanks!

/Hristo Stoyanov
On Jan 4, 2016 9:20 AM, "Max Barkley" <notifications@github.com> wrote:

> Hristo,
> I've squashed your two commits and cherry-picked them to master
> <https://github.com/errai/errai/commit/306e94eb2580b2b53bb6c7819f83d12e47fc13ff>
> and 3.2
> <https://github.com/errai/errai/commit/941629a7894aa8460abac6e7f1d02d62e7585c8e>
> .
> Cheers.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/errai/errai/pull/153#issuecomment-168741324>.
>
